lib/tld.js

```diff
@@ -141,7 +141,7 @@ tld.prototype.getRulesForTld = function getRulesForTld (tld, default_rule) {
   }
 
   // Nothing found, apply some default value
-  if (!rules) {
+  if (rules === void 0) {
```

Comment:
You are right @GreyKn, it seems that I am not familiar anymore with my own codebase ;-)

I now understand why the `if` statement check is problematic, sorry if it has been a pain to explain!